tests/modules/test_definegrid.py

```diff
@@ -29,53 +29,6 @@
 
 
 class TestDefineGrid(unittest.TestCase):
-    def test_01_02_load_v1(self):
```

Comment:
What's the reason for removing this test?

Reply:
There was some difficulty fixing the load_v1 tests so after asking, Allen suggested removing them

Reply:
Makes sense then! If you guys can keep those conversations in the ip-cellprofiler channel, we will have access to them :)